src/components/folder/CreateFolder.tsx

```diff
@@ -53,7 +53,7 @@ const CreateFolder = (): JSX.Element => {
 			],
 		};
 
-		const created = await user.contact.createFolder(folder);
+		const created = await user.fullContact.folders.createFolder(folder);
```

Comment:
```suggestion
		const created = await user.fullContact.folders.create(folder);
```
You can rename the methods to avoid repeating `folder`, `file`, `program` etc now that you have dedicated classes